x11-plugins/gkrellm-leds/gkrellm-leds-0.8.2-r2.ebuild

```diff
@@ -30,7 +30,7 @@ src_prepare() {
 	eautoreconf
 }
 
-src_configure() {
+src_install() {
 	PLUGIN_SO=( src/.libs/gkleds$(get_modname) )
 	default
```

Comment:
gkrellm-plugin.eclass overrides src_install, should we call gkrellm-plugin_src_install instead of default here?

Reply:
Yep!